src/test/java/uk/ac/ebi/eva/pipeline/jobs/GenotypedVcfJobTest.java

```diff
@@ -65,12 +66,15 @@
     @Autowired
     private JobLauncherTestUtils jobLauncherTestUtils;
 
-    private static String opencgaHome = System.getenv("OPENCGA_HOME") != null ? System
-            .getenv("OPENCGA_HOME") : "/opt/opencga";
+    private static String opencgaHome = GenotypedVcfJobTestUtils.getDefaultOpencgaHome();
```

Comment:
This variable is not needed any more I think. Also applies to `GenotypedVcfJobWorkflowTest`.